tensorflow/core/util/mkl_util.h

```diff
@@ -1245,7 +1652,13 @@ class MklDnnData {
     CHECK_NOTNULL(user_memory_);
     return op_pd != user_memory_->get_primitive_desc();
   }
+#endif
 
+#ifndef ENABLE_MKLDNN_V1
+  /// In MKL-DNN v1.0, it it is not possible to directly compare two memory
```

Comment:
```suggestion
  /// In MKL-DNN v1.x, it it is not possible to directly compare two memory
```
Please replace all `v1.0`s in the comment with `v1.x`.

Reply:
Done.